CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * None.
 
 ### Fixes
+* [SyncConfiguration] The `SyncConfiguration.Builder.allowQueriesOnUiThread` flag was wrongly initialized to `false` keeping users from running queries from the UI thread when using synced Realms. It now defaults to `true`, allowing queries to be run from the UI. See [#7177](https://github.com/realm/realm-java/issues/7177)
```

Comment:
```suggestion
* [SyncConfiguration] The `SyncConfiguration.Builder.allowQueriesOnUiThread` flag was wrongly initialized to `false` keeping users from running queries from the UI thread when using synced Realms. It now defaults to `true`, allowing queries to be run from the UI. (Issue [#7177](https://github.com/realm/realm-java/issues/7177), since 10.0.0)
```